tests/test_execution_compute_plan.py

```diff
@@ -331,7 +342,8 @@ def test_execution_compute_plan_canceled(global_execution_env):
     assert cp.status == assets.Status.canceled
 
     cp = cp.future().wait()
-    assert cp.status == assets.Status.canceled
+    assert cp.status in assets.Status.canceled
```

Comment:
This will work by why the change here?

Reply:
Good catch. Looks like an invalid syntax.